src/Landing/redux/actions/LatestArticles.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import { FETCH_LATEST_POSTS } from '../types';
+
+const fetchLatestArticles = () => (dispatch) => {
+  axios.get('https://ah-backend-realers-staging.herokuapp.com/api/articles/?format=json&limit=9')
+    .then(res => dispatch({
+      type: FETCH_LATEST_POSTS,
+      payload: res.data.results,
+    }))
+    .catch((error) => {
+      console.error('Error fetching articles', error);
+    });
+};
+
+export default fetchLatestArticles;
```

Comment:
This file could use a better name like `latestArticlesActions.js`

Reply:
I am done renaming.